### PR TITLE
BDLaethg47

### PR DESCRIPTION
**Pull request by thea-m**
Here is my last scroll :)

### CODE REVIEW

File: OxfordBodleian/BodNewMSS/BDLaethg47.xml [hunk 1]
```diff
@@ -50,45 +141,145 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                         </objectDesc>
                         
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+                                <date notBefore="1800" notAfter="2020"/>
+                                <desc>Trained hand. Bulky, widely spaced characters.</desc>
+                           <seg type="ink">Black, red.</seg>
+                                <seg type="rubrication">The first two–three lines of prayers; holy names; the complete name of the owner or only the holy name contained in it; 
+                                <foreign xml:lang="gez">ሰላም፡ ለከ፡</foreign>/<foreign xml:lang="gez">ለኪ፡</foreign>; parts of numerals. In the final supplication, in the illegible
```
Comment (MassimoVilla) [inline OxfordBodleian/BodNewMSS/BDLaethg47.xml:215]: perhaps one "in" is redundant?

File: OxfordBodleian/BodNewMSS/BDLaethg47.xml [hunk 2]
```diff
@@ -129,15 +323,88 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                <persName ref="PRS14043WalattaS">ወለተ፡ <hi rend="rubric">ሥላሴ፡</hi></persName>
+                                <hi rend="rubric"><gap reason="illegible" unit="chars" quantity="1"/>ር፡</hi>
+                            </ab>
+                        </div>
+                        <div type="textpart" subtype="text" n="5" corresp="#ms_i4" xml:id="text4">
+                            <ab>
+                                <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩
+                                    አምላክ<supplied reason="omitted">፡</supplied> ጸሎት፡ በእንተ፡ ሕማመ<supplied reason="omitted">፡</supplied></hi>
+                                ሸቶላይ፡ አንተ፡ ሸቶላይ፡ ወሸቶላዊ፡ ጋኔን፡ ዘ<sic>ትቀት<supplied reason="omitted">፡</supplied></sic> ሕፃናተ፡ ወትቋርጽ፡ አማሉተ፡ ወትጦዊ፡ አንጀተ፡
+                                ወትቆረጥም፡ አካላተ፡ ወተሐምም፡ አባላተ፡ አምሐልኩከ፡ ወአውገዝኩከ፡ ወለጐምኩከ፡ ወአስርኩከ፡ ወቆለፍኩከ፡ ወሐተምኩከ፡ በማሕተመ<supplied reason="omitted">፡</supplied>
```
Comment (MassimoVilla) [inline OxfordBodleian/BodNewMSS/BDLaethg47.xml:394]: Again አሰርኩከ፡ ?

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you Thea, just very few observations, two of which related to the Work record

**Comment by thea-m** [comment]
Thank you, Massimo!

**Review by MassimoVilla — APPROVED** [review]

**Review by ElianaDalSasso — APPROVED** [review]
Thanks Thea!

**Pull request merged**